phx/events/models.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from django.contrib.auth.models import User
+from django.db import models
+
+logger = logging.getLogger(__name__)
+
+
+class Event(models.Model):
```

Comment:
On reflection perhaps Events and Performances should be grouped under Results :thinking:

Reply:
Moved everything around in this PR: https://github.com/orangespaceman/phx/pull/89

Reply:
Yes, agreed. It's all related to Results so makes sense to keep them together 👍